Fixes section links for user-interface/images.md

michaelspiss opened the pull request:
Section links were not working due to different names/capitalization.

michaelspiss committed:
Fix section links for user-interface/images.md

Section links were not working due to different names/capitalization.

conceptdev commented:
Thanks for the update. Because of the anchor change, we'll need to check there aren't other inbound links to this page using the old anchor. The PR will be merged, just need a little time to validate.

conceptdev committed:
minor update: use auto-generated anchor instead

Note that this anchor is also referenced from xamarin-forms/xaml/markup-extensions/creating.md so that will be updated manually in another PR

conceptdev reviewed (approved):
thanks for noticing the broken anchor and fixing. i'll be making an additional change to sync this up with another page's inbound link.
